cdc/sink/sink.go

```diff
@@ -53,7 +53,7 @@ type Sink interface {
 	//
 	// FlushRowChangedEvents is thread-safe.
 	// FIXME: some sink implementation, they should be.
-	FlushRowChangedEvents(ctx context.Context, tableID model.TableID, resolvedTs uint64) (uint64, error)
+	FlushRowChangedEvents(ctx context.Context, tableID model.TableID, resolvedTs uint64) (uint64, bool, error)
```

Comment:
the FlushRowChangedEvents func is already non-blocking

Reply:
1. Indeed it is blocking in current`mqSink` implementation.
2. We shoul make this interface method explicit non-blocking rather than implicit non-blokcing that hide in the implementation to make it easier to use.